app/views/search/_granule_list.html.erb

```diff
@@ -84,15 +84,15 @@ Delimiters: Separate multiple granule IDs by space, comma, or new line."/>
 
               <div class="granule-temporal-range">
                 <!-- ko with: getTemporal() -->
-                  <!--ko if: ($data[0] && !$data[1])-->
                   <div class="temporal-start"><h5>START</h5><p data-bind="text: $data[0]"></p></div>
-                  <!--/ko-->
-                  <!--ko if: ($data[1] && !$data[0])-->
-                  <div class="temporal-end"><h5>END</h5><p data-bind="text: $data[1]"></p></div>
-                  <!--/ko-->
-                  <!--ko if: ($data[0] && $data[1])-->
-                  <div class="temporal-start"><h5>START</h5><p data-bind="text: $data[0]"></p></div><div class="temporal-end"><h5>END</h5><p data-bind="text: $data[1]"></p></div>
-                  <!-- /ko -->
+                  <div class="temporal-end"><h5>END</h5>
+                    <!--ko if: $data[1]-->
+                    <p data-bind="text: $data[1]"></p>
```

Comment:
This can be much simpler just using `<p data-bind="text: $data[1] || Not Provided"></p>` or some variation using a method.